Kernel_d/doc/Kernel_d/CGAL/Kernel_d/Vector_d.h

```diff
@@ -96,8 +96,8 @@ InputIterator first, InputIterator last);
 introduces a 
 variable `v` of type `Vector_d<Kernel>` in dimension `d` 
 initialized to the vector with homogeneous coordinates as defined by 
-`H = set [first,last)` and `D`: \f$ (\pmH[0], 
-\pmH[1], \ldots, \pmH[d-1], \pmD)\f$. The sign 
+`H = set [first,last)` and `D`: \f$ (\pm H_0, 
```

Comment:
I think the correct fix here is to use backticks.

Reply:
I made it conform the lines in the comment block above, but it might be that the intention was backticks.

Reply:
Note also that `\pm` is the plus/minus sign and in backticks this has to be something different as well.
Some possibilities are:
  - ```\f$ \pm H[0] \f$```
  -  ```&plusmn;H[0]```
  -  ``` `&plusmn;H[0]` ```